cmd/machine-config-operator/start.go

```diff
@@ -42,7 +43,7 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	flag.Parse()
 
 	// To help debugging, immediately log version
-	glog.Infof("Version: %+v (%s)", version.Raw, version.Hash)
+	glog.Infof("Version: %s (Raw: %s, Hash: %s)", os.Getenv("RELEASE_VERSION"), version.Raw, version.Hash)
```

Comment:
So, if I understand this correctly `os.Getenv("RELEASE_VERSION")` is just getting an environment variable. In this case nothing is setting `RELEASE_VERSION` no? I logged in to a node and can see that its empty by default. Wouldn't this just be empty?